packages/hoppscotch-common/src/components/importExport/ImportExportSteps/FileImport.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="space-y-4">
+    <p class="flex items-center">
+      <span
+        class="inline-flex items-center justify-center flex-shrink-0 mr-4 border-4 rounded-full border-primary text-dividerDark"
+        :class="{
+          '!text-green-500': hasFile,
+        }"
+      >
+        <icon-lucide-check-circle class="svg-icons" />
+      </span>
+      <span>
+        {{ t(`${caption}`) }}
+      </span>
+    </p>
+    <p
+      class="flex flex-col ml-10 border border-dashed rounded border-dividerDark"
+    >
+      <input
+        id="inputChooseFileToImportFrom"
+        ref="inputChooseFileToImportFrom"
+        name="inputChooseFileToImportFrom"
+        type="file"
+        class="p-4 cursor-pointer transition file:transition file:cursor-pointer text-secondary hover:text-secondaryDark file:mr-2 file:py-2 file:px-4 file:rounded file:border-0 file:text-secondary hover:file:text-secondaryDark file:bg-primaryLight hover:file:bg-primaryDark"
+        :accept="acceptedFileTypes"
+        @change="onFileChange"
+      />
+    </p>
+
+    <div>
+      <HoppButtonPrimary
+        class="w-full"
+        :label="t('import.title')"
+        :disabled="!hasFile"
+        @click="emit('importFromFile', fileContent)"
+      />
+    </div>
```

Comment:
Am I right in assuming the role of the `div` wrapper here is to provide a block-level element behavior?